static/js/common.js

```diff
@@ -20,16 +20,16 @@ function password_quality(password, bar, password_field) {
         return undefined;
     }
 
-    var min_length = 6;
+    var min_password_length = 1;
```

Comment:
Also, we should uncomment 
# PASSWORD_MIN_LENGTH = 6
in `zproject/prod_settings_template` as a part of this